examples/Nat/TcpNAT.cs

```diff
@@ -0,0 +1,72 @@
+using PacketDotNet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Net.NetworkInformation;
+using System.Net;
+
+namespace Pax.Examples.Nat
+{
+  /// <summary>
+  /// A class that handles TCP packets and performs Port-Address Translation.
+  /// </summary>
+  public sealed class TcpNAT : NATBase<TcpPacket, NodeWithPort, TcpPacketEncapsulation>
+  {
+    /// <summary>
+    /// The duration of the TCP TIME-WAIT TIMEOUT. See RFC 793.
+    /// </summary>
+    private readonly TimeSpan TIME_WAIT;
+
+    /// <summary>
+    /// The start of the range of TCP ports to use (inclusive).
+    /// </summary>
+    private readonly ushort StartPort;
+
+    /// <summary>
+    /// The end of the range of TCP ports to use (inclusive).
+    /// </summary>
+    private readonly ushort EndPort; // FIXME we don't use EndPort yet
```

Comment:
Since `EndPort` isn't being used, am I correct in thinking that currently (in theory) assigned port numbers could reach the maximum value of `ushort`?